SPECS/pciutils/pciutils.spec

```diff
@@ -47,7 +47,6 @@ chmod -v 766 %{buildroot}%{_libdir}/libpci.so
 %files
 %defattr(-,root,root)
 %{_sbindir}/*
-%{_libdir}/*.so.*
```

Comment:
The changes look good to me but since we're here, would you mind changing the `%files libs` section to use `%{_libdir}/libpci.so.3.*` instead of `%{_libdir}/libpci.so.*`? The full file path is `/usr/lib/libpci.so.3.7.0`, so I think it should be fine.

Reply:
Change is done and tested, thanks!